graylog2-web-interface/config.js

```diff
@@ -23,4 +23,7 @@ window.appConfig = {
     'org.graylog.plugins.customization.notifications': {},
   },
   isCloud: false,
+  featureFlags: {
+    'legacy-aggregation-wizard': 'on',
```

Comment:
We should also add the legacy-aggregation-wizard feature flag to the feature-flag.config file and document it.

Reply:
Thanks, @AntonEbel I will do that. I will also remove it from the config file this was a test.